### ISSUE TITLE
Include Play Store's required Data Safety guidance documentation

### ISSUE THREAD

**Issue by doug-textnow** [jitsi/jitsi-meet#11608]
**Is your feature request related to a problem you are facing?**  Yes.
Developers who publish apps to the Google Play Store are required to disclose whether their app collects or shares a variety of types of data (see: https://support.google.com/googleplay/android-developer/answer/10787469)  This includes data that is collected or shared by embedded SDKs such as Jitsi Meet.  It is difficult, time-consuming, and wasteful to have every app author conduct their own investigation to determine which of the designated categories of data Jitsi-Meet collects, and for which purposes.  

**Describe the solution you'd like**
Keep up-to-date documentation on the data collected by the Android Jitsi Meet SDK.  Structure document to match the [categories](https://support.google.com/googleplay/android-developer/answer/10787469#zippy=%2Cdata-types) and [purposes](https://support.google.com/googleplay/android-developer/answer/10787469#zippy=%2Cdata-types%2Cpurposes) defined by the Play Store.  This will allow app publishers to quickly and easily add the answers from the Jitsi Meet documentation to their app's Data Safety form, and facilitate wider adoption of Jitsi.  

**Describe alternatives you've considered**
Add Jitsi Meet to Google's SDK index: https://play.google.com/sdks and include Data Safety

**Comment by damencho** [comment]
Nothing is being collected from the SDK if not configured. So everything depends on the deployment it will be used with, whether analytics are enabled and what is collected depends on its implementation, is callstats enabled or not ... so everything depends on the usage of the SDK and the deployment it is used with.

**Comment by doug-textnow** [comment]
That's good to know!  The documentation should probably then be written in a way that makes clear the mapping between each item in the configuration and the types of data that will be collected (along with their purposes) as a result.

**Comment by damencho** [comment]
This really depends on what you have enabled and for the services you enable you can consult their docs:
https://www.callstats.io/terms
https://amplitude.com/terms
If you have enabled any other analytics on your deployment consult their documentation.

**Comment by doug-textnow** [comment]
I understand that some of the data leaving the SDK may be optional, or conditional - that's completely on par with other SDKs, in fact, it's probably _less_ common for an SDK to collect the exact same info from every user on every app it's embedded it.  _Most_ SDKs have _some_ degree of configurability or environment-dependent variation, and this doesn't preclude them from providing documentation that gives app authors the information they need to be able to comply with Play Store policies.  

The Play Store requires authors to answer and submit the Data Safety form even if every single type of data identified by the Store's policy is not collected.  Even in the case of an SDK that makes no network calls and transmits no data off the device, App authors still have a need for SDK providers to provide them with official documentation to that effect.  It's not realistic to expect every user of the SDK to have the intimate familiarity with all parts of the codebase which would be required to make a confident assertion that the SDK does not collect a certain type of data.  

In the case of Jitsi, some data _has_ to be collected (ip for example) and other data may depend on external factors.    The data types that are necessarily collected should be disclosed, and the optionally or inconsistently collected data types should also be disclosed with clarification about the dependent factors attached.  

To avoid any misunderstanding or ambiguity around the meaning of "collection" or other vocabulary, I'll quote from the Play Store policy:

> “Collect” means transmitting data from your app off a user’s device. Please note the following guidelines:
> 
>     Libraries and SDKs: This includes user data transmitted off device from your app by libraries and/or SDKs used in your app, irrespective of whether data is transmitted to you or a third-party server.
>     Webview: It also includes user data collected from a webview which has been opened from your app, if your app is in control of the code/behavior delivered through that webview. 
>         You do not need to declare data collection from a webview in which users are navigating the open web.
>     Ephemeral processing: User data transmitted off device that is processed ephemerally needs to be included in your form response, but if it meets the standard below, it will not be disclosed in your app’s Data safety section on Google Play.
>         Processing data “ephemerally” means accessing and using it while the data is only stored in memory and retained for no longer than necessary to service the specific request in real-time.
>         For example, a weather app that transmits user location off the device to fetch the current weather at the user's location but only uses location data in memory and does not store that data once the request has been fulfilled, can treat its transient use of location as ephemeral. However, using data to build advertising profiles or other user profiles cannot be treated as ephemeral and must be declared as collection or sharing for the relevant purposes.
>     Pseudonymous data: User data collected pseudonymously must be disclosed. For example, data that can reasonably be re-associated with a user must be disclosed.

Note that data does not need to be stored on a server long-term to meet the definition of "collection", and even ephemerally processed data must be disclosed.

For anyone who has not filled out the Play Store's Data Safety form for an app before, I highly encourage going through the process a couple times, and really thinking about it from the perspective of someone who wants or _needs_ to comply with the policy in letter and spirit.  I believe anyone who does so will appreciate the utility of the documentation being discussed here.

**Comment by saghul** [comment]
Hey @doug-textnow I just filled in the data safety section. I must have missed the notification, as I didn't see any banner or anything in the Play console.

We had to file a similar thing for the Apple Store not that long ago.

It should show up on the Play Store soon, but here is the preview:

<img width="2672" alt="Screenshot 2022-06-03 at 10 00 56" src="https://user-images.githubusercontent.com/317464/171813684-3b0c26e8-75b1-42a0-91ab-178f11d3ec63.png">

Cheers!
